packages/sdk/src/contracts/prebuilt-implementations/marketplace.ts

```diff
@@ -296,6 +296,39 @@ export class Marketplace implements UpdateableNetwork {
     return this.contractWrapper.readContract.timeBuffer();
   }
 
+  /**
+   * Get all the offers
+   *
+   * @remarks Fetch all the offers for a specified direct listing.
+   * @example
+   * ```javascript
+   * const offers = await marketplaceContract.getOffers(listingId);
+   * const firstOffer = offers[0];
+   * ```
+   *
+   * @param listingId - the id of the listing to fetch offers for 
+   */
+   public async getOffers(
+    listingId: BigNumberish,
+  ): Promise<(Offer)[]> {
```

Comment:
you don't need the parenthesis around Offer here. Would you mind setup prettier so it applies the common formatting for you? (in vscode, settings, default formatter > prettier)